packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1191,6 +1191,23 @@ enum ResizeImagePolicy {
 /// Instructs Flutter to decode the image at the specified dimensions
 /// instead of at its native size.
 ///
+/// The [width] and [height] parameters refer to logical pixels.
+///
+/// Logical pixels are roughly the same visual size across devices.
+/// Physical pixels are the size of the actual hardware pixels on the device.
+/// The number of physical pixels per logical pixel is described by the `devicePixelRatio`.
```

Comment:
What do you think about using `[FlutterView.devicePixelRatio]` here?

Reply:
Since `FlutterView` is defined in `dart:ui`, which is imported in https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/painting/image_provider.dart#L11

this would need to be `[ui.FlutterView.devicePixelRatio]`, right?